website/docs/faqs/Accounts/change-billing.md

```diff
@@ -6,6 +6,6 @@ id: change-billing
 ---
 
 
-If you want to change your account's credit card details, select the gear menu in the upper right corner of dbt Cloud. Go to Account Settings  &#8594; Billing  &#8594; Payment Information. Enter the new credit card details on the respective fields then click on **Update payment information**. Only the _account owner_ can make this change. 
+If you want to change your account's credit card details, go to the left side panel, click Account Settings  &#8594; Billing  &#8594; Payment Information. Enter the new credit card details on the respective fields then click on **Update payment information**. Only the _account owner_ can make this change. 
```

Comment:
let's bold the ui elements/pages since we're here and switch the sentence case to be consistent with what in the UI
```suggestion
If you want to change your account's credit card details, go to the left side panel, click **Account settings**  &#8594; **Billing**  &#8594; scroll to **Payment information**. Enter the new credit card details on the respective fields then click on **Update payment information**. Only the _account owner_ can make this change. 
```